ci/Jenkinsfile.tests-e2e

```
@@ -76,6 +76,8 @@ pipeline {
 
     TESTRAIL_URL = 'https://ethstatus.testrail.net'
     TESTRAIL_PROJECT_ID = 17
+    /* Override QT plugin with linux */
+    QT_QPA_PLATFORM = "linuxfb"
```

We need to be more explicit:
```suggestion
    /* Override QT xcb plugin with linux to avoid errors like:
     * "Could not load the Qt platform plugin "xcb" in "" even though it was found." */
    QT_QPA_PLATFORM = "linuxfb"
```

---

addressed, thank you.